solutions/src/main/java/ru/sberbank/school/task06/CountMapImpl.java

```diff
@@ -0,0 +1,53 @@
+package ru.sberbank.school.task06;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+public class CountMapImpl implements CountMap<String> {
+
+    private Integer count;
+    private Map<String,Integer> countHashMap;
+    CountMapImpl(){
```

Comment:
&#39;CTOR_DEF&#39; should be separated from previous statement.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-6696297>*